{
    "source file": "test_nmf.py",
    "line number": "542",
    "func name": "test_nmf_custom_init_dtype_error",
    "func arg": "()",
    "comments": ""
}